server/src/main/java/server/haengdong/presentation/request/BillActionsSaveRequest.java

```diff
@@ -0,0 +1,13 @@
+package server.haengdong.presentation.request;
+
+import java.util.List;
+import server.haengdong.application.request.BillActionAppRequest;
+
+public record BillActionsSaveRequest(List<BillActionSaveRequest> actions) {
```

Comment:
```suggestion
public record BillActionsSaveListRequest(List<BillActionSaveRequest> actions) {
```
a: 저희 페어는 위와 같이 네이밍을 했어요. 같이 고민해 보아요.

Reply:
List로 자료형을 명시하는 것보단 s로 복수형을 표현하는 것이 간결해서 보기 좋은 것 같아요.